packages/sanity/src/structure/comments/src/components/list/CommentsListItemLayout.tsx

```diff
@@ -298,28 +310,58 @@ export function CommentsListItemLayout(props: CommentsListItemLayoutProps) {
         <Flex align="center" gap={FLEX_GAP} flex={1}>
           <CommentsAvatar user={user} />
 
-          <Flex align="center" paddingBottom={1} sizing="border" flex={1}>
-            <Flex align="flex-end" gap={2}>
-              <Box flex={1}>{name}</Box>
-
-              {!displayError && (
-                <Flex align="center" gap={1}>
-                  <TimeText muted size={0}>
-                    <time dateTime={createdDate.toISOString()} title={formattedCreatedAt}>
-                      {createdTimeAgo}
-                    </time>
-                  </TimeText>
-
-                  {formattedLastEditAt && editedDate && (
-                    <TimeText muted size={0} title={formattedLastEditAt}>
-                      <time dateTime={editedDate.toISOString()} title={formattedLastEditAt}>
-                        ({t('list-item.layout-edited')})
+          <Flex direction="column" gap={2} paddingY={comment.context?.intent ? 2 : 0}>
+            <Flex
+              align="center"
+              paddingBottom={comment.context?.intent ? 0 : 1}
+              sizing="border"
+              flex={1}
+            >
+              <Flex align="flex-end" gap={2}>
+                <Box flex={1}>{name}</Box>
+
+                {!displayError && (
+                  <Flex align="center" gap={1}>
+                    <TimeText muted size={0}>
+                      <time dateTime={createdDate.toISOString()} title={formattedCreatedAt}>
+                        {createdTimeAgo}
                       </time>
                     </TimeText>
-                  )}
-                </Flex>
-              )}
+
+                    {formattedLastEditAt && editedDate && (
+                      <TimeText muted size={0} title={formattedLastEditAt}>
+                        <time dateTime={editedDate.toISOString()} title={formattedLastEditAt}>
+                          ({t('list-item.layout-edited')})
+                        </time>
+                      </TimeText>
+                    )}
+                  </Flex>
+                )}
+              </Flex>
             </Flex>
+
+            {comment.context?.intent && (
+              <Box flex={1}>
+                <IntentText muted size={0} textOverflow="ellipsis" title={''}>
```

Comment:
Is this `title={''}` needed?

Reply:
Not as far as I know! Thanks for spotting.